aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -246,6 +248,38 @@ module aptos_framework::block {
         };
     }
 
+    /// `block_prologue()` but trigger reconfiguration with DKG after epoch timed out.
```

Comment:
1. Should this comment start with "`block_prologue_**ext**()` but [...]"
2. Can you add a comment to the old `block_prologue_ext()` to indicate when it is called and when it isn't? Same to this function?